lib/rutema/core/engine.rb

```diff
@@ -200,7 +321,12 @@ def exit
         Thread.kill(@thread)
       end
     end
+
     private
+
+    ##
+    # As long as the incoming queue is not empty dispatch its messages to all
```

Comment:
Dispatch messages from the incoming queue to all subscribers until the queue is empty.

Reply:
Applied you suggestion :+1: